src/DistributionCenter.Application/Tables/Core/Bases/BaseJsonTable.cs

```diff
@@ -0,0 +1,36 @@
+namespace DistributionCenter.Application.Tables.Core.Bases;
+
+using Components.Information.Interfaces;
+using Components.QueryCommands.Concretes.File.Concretes;
+using Components.QueryCommands.Interfaces;
+using Connections.File.Interfaces;
+using Domain.Entities.Interfaces;
+using Interfaces;
+
+public abstract class BaseJsonTable<T>(IFileConnectionFactory<T> fileConnectionFactory) : ITable<T>
```

Comment:
This class should be BaseFileTable instead of BaseJsonTable.

Reply:
Good observation, i fix it